src/app/zap-templates/zcl/data-model/chip/valve-configuration-and-control-cluster.xml

```diff
@@ -0,0 +1,97 @@
+<?xml version="1.0"?>
+<!--
+Copyright (c) 2023 Project CHIP Authors
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+-->
+<configurator>
+  <domain name="CHIP"/>
+
+  <bitmap name="Feature" type="bitmap32">
+    <cluster code="0x0081"/>
+    <field name="TimeSync" mask="0x1"/>
+    <field name="Level" mask="0x2"/>
+  </bitmap>
+
+  <bitmap name="ValveFaultBitmap" type="bitmap16">
+    <cluster code="0x0081"/>
+    <field name="GeneralFault" mask="0x1"/>
+    <field name="Blocked" mask="0x2"/>
+    <field name="Leak" mask="0x4"/>
```

Comment:
spec names this `Leaking`

Reply:
This was changes somewhat last minute on the spec side, but yes this is now "Leaking"